webview/src/experiments/components/table/Table.test.tsx

```diff
@@ -258,6 +258,23 @@ describe('Table', () => {
     })
   })
 
+  describe('Head Depth', () => {
+    it('should be updated by the user in the header context menu', async () => {
```

Comment:
Do we need more than this test for this feature?

Reply:
"Backend" -> https://github.com/iterative/vscode-dvc/blob/558c09e09947ef755067aec509e7fefafb4aa4ba/extension/src/test/suite/experiments/index.test.ts#L172